src/main/java/org/kohsuke/github/GHCreateTeamBuilder.java

```diff
@@ -0,0 +1,92 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Creates a team.
```

Comment:
Add a link to the github api documentation for create team

Reply:
Done